splitio/client/factory.py

```diff
@@ -393,10 +398,15 @@ def _build_redis_factory(api_key, cfg):
         _LOGGER.warning("dataSampling cannot be less than %.2f, defaulting to minimum",
                         _MIN_DEFAULT_DATA_SAMPLING_ALLOWED)
         data_sampling = _MIN_DEFAULT_DATA_SAMPLING_ALLOWED
+
+    imp_strategy = StrategyOptimizedMode(Counter()) if cfg['impressionsMode'] == 'OPTIMIZED' else StrategyDebugMode()
```

Comment:
this line needs to be commented for now because we will default to StrategyDebugMode until we implement all the logic for redis. The line is ok, but just defaulting to Strategy until we do that

Reply:
done